client/jetpack-cloud/sections/partner-portal/payment-method-form/index.tsx

```diff
@@ -140,6 +140,9 @@ function PaymentMethodForm() {
 
 	const handleChangeError = useCallback(
 		( { transactionError }: { transactionError: string | null } ) => {
+			if ( !! transactionError && transactionError.includes( 'CVC' ) ) {
```

Comment:
We should be doing `transactionError.includes( 'cvc' )`

```suggestion
			if ( transactionError && transactionError.includes( 'cvc' ) ) {
```